invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,3 +28,49 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task(ignore_result=True)
+def update_expired_embargos():
+    """Release expired embargoes every midnight."""
+
+    def lift_embargo_from_draft(service, rdm_record):
+        """Lifts embargo from draft."""
+        record_draft = service.edit(rdm_record["id"], system_identity)
```

Comment:
**NOTE**: You can ignore this comment, im just leaving it here to showcase my confusion with the naming 😅 

moderate: `edit` [creates a new draft/revision out of a record](https://github.com/inveniosoftware/invenio-drafts-resources/blob/master/invenio_drafts_resources/services/records/service.py#L216), if the draft exists it returns it (which is what's happening). Then the code is commiting directly the `_record` object.

Could this be achieved using the [`service.update_draft`](https://github.com/inveniosoftware/invenio-drafts-resources/blob/master/invenio_drafts_resources/services/records/service.py#L163)? That way we avoid accessing the data access layer, and we only interact with the service layer.